storybook/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -20,22 +20,30 @@ const meta = {
     disabled: false,
   },
   argTypes: {
+    cols: {
+      control: {
+        type: 'number',
+      },
+      description: 'The maximum number of characters per line.',
```

Comment:
This description doesn't read to me as if it's about the width of the Text Area.

How about:

```suggestion
      description: 'The width in avarage character widths.',
```

Reply:
True. I’ve made it ‘The width, expressed in the average number of characters’.